res/PyTorchExamples/examples/strided_slice/__init__.py

```diff
@@ -0,0 +1,22 @@
+import torch
+import torch.nn as nn
+
+
+# model
+class net_strided_slice(nn.Module):
+    def __init__(self, begin, end, stride):
+        super().__init__()
+        self.key = [slice(begin[i], end[i], stride[i]) for i in range(len(begin))]
+
+    def forward(self, input):
+        # this is general way to do input[:, :, 1:5:2, 0:5:2]
+        return input[self.key]
+
+    def onnx_opset_version(self):
+        return 10
```

Comment:
opset version 9 does not support strides, so we need at least 10-th version.

Used in #8000